src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -18,6 +20,8 @@ public class ClearCommand extends Command {
     public CommandResult execute(Model model) {
         requireNonNull(model);
         model.setAddressBook(new AddressBook());
+        model.setAssignments(new AssignmentList());
+        model.setTutorials(new TutorialList());
```

Comment:
I like how you implement the clear command! It resets the data for tutorial and assignment!